scripts/src/env/start.rs

```diff
@@ -26,15 +28,28 @@ pub async fn execute() -> anyhow::Result<Environment> {
 
     println!("✓");
 
+    print_progress!("Reading config file");
+
+    let path = std::env::current_dir()?.join(Path::new(config::FILE_NAME));
+    let config = match Config::from_file(&path) {
+        Ok(config) => (config.ethereum, config.bitcoin),
+        Err(e) => {
+            eprintln!("Could not load config from file: {}", e.to_string());
+            (None, None)
+        }
+    };
+
+    println!("✓");
```

Comment:
We are printing the tick here even if we fail to read the config file, I guess this should move into the `Ok` branch.